js/chat_app/src/redux/auth/authOperation.js

```diff
@@ -0,0 +1,12 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import login from '../../services/services';
+
+const loginAuth = createAsyncThunk('auth/login', async ({ username, password }) => {
+  const response = await login({ username, password });
+  const data = await response.json();
+  const result = await localStorage.setItem('auth', JSON.stringify(data));
```

Comment:
it's not async operation. remove await

Reply:
and i belive that you can just do `localStorage.setItem('auth', JSON.stringify(data));`